<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textbox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据保存</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>目标值设置</string>
       </property>
       <widget class="QPushButton" name="speedSend">
        <property name="geometry">
         <rect>
          <x>146</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>36</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>速度：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>51</x>
          <y>20</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>20</y>
          <width>18</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>rpm</string>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>0</width>
          <height>23</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>36</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>位置：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="posSend">
        <property name="geometry">
         <rect>
          <x>146</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>50</y>
          <width>18</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>圈</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_5">
        <property name="geometry">
         <rect>
          <x>51</x>
          <y>50</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>增量值设置</string>
       </property>
       <widget class="QPushButton" name="deltaSpeendSend">
        <property name="geometry">
         <rect>
          <x>146</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>36</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>速度：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="geometry">
         <rect>
          <x>51</x>
          <y>20</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>20</y>
          <width>18</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>rpm</string>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>0</width>
          <height>23</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>36</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>位置</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deltaPosSend">
        <property name="geometry">
         <rect>
          <x>146</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>50</y>
          <width>18</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>圈</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_7">
        <property name="geometry">
         <rect>
          <x>51</x>
          <y>50</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-10</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>271</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>转速：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>扭矩：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>电流：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>转角：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>摆速：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>温度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="tingji">
       <property name="text">
        <string>停机</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="duankai">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="fuwei">
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tuichu">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
